docs/reference/api/drivers/RawSerial.md

```diff
@@ -1 +1,34 @@
 ## RawSerial
+
+The RawSerial class provides UART functionality without the use of [Stream's](https://github.com/ARMmbed/Handbook/blob/new_engine/docs/reference/api/platform/Stream.md)  print and scan functions as in the Serial class. Instead of using the standard library print and scan functions they are re-implemented to work using each target's underlying serial communication functions. See the porting guide for [target serial support](https://os.mbed.com/docs/v5.6/reference/contributing-target.html#serial). This makes it suitable for use in interrupt handlers with the RTOS.
```

Comment:
I don't think I understand the second sentence. Who or what uses the standard library print and scan functions? Who or what is reimplementing the print and scan functions to work using each target's underlying serial communication functions? Also, is Stream part of the Serial class?

Reply:
Sorry sort of wrote that after looking through Serial and RawSerial implementations for a while and it made sense in my head at the time. I'll rephrase it.

Reply:
Updated it to hopefully be more clear, let me know if it's not. 

The Serial class inherits from the Stream class, yes. Serial uses the Stream class's print and scan functions as a child.